skit.Infrastructure/skit.Infrastructure.csproj

```diff
@@ -8,11 +8,14 @@
 
     <ItemGroup>
       <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.5.1" />
+      <PackageReference Include="MailKit" Version="4.3.0" />
```

Comment:
MailKit to jest jakiś provider do wysyłania maili? Czy tylko jakaś biblioteczka do łatwiejszego obsługiwania tego wysyłania

Reply:
Biblioteka, bazowo .net ma tam swoje narzędzie, ale nawet Microsoft poleca by używać tego XD